docker/test/integration/minifi/core/Container.py

```diff
@@ -0,0 +1,102 @@
+import docker
+import logging
+import tarfile
+from io import BytesIO
+
+
+class Container:
+    def __init__(self, name, engine, vols, network):
+        self.name = name
+        self.engine = engine
+        self.vols = vols
+        self.network = network
+
+        # Get docker client
+        self.client = docker.from_env()
+        self.image = None
+        self.deployed = False
+
+    def __del__(self):
+        self.cleanup()
+
+    def cleanup(self):
+        logging.info('Cleaning up container: %s', self.name)
+        try:
+            self.client.containers.get(self.name).remove(v=True, force=True)
+        except docker.errors.NotFound:
```

Comment:
Just wondering if it should log anything over here regarding the container cleanup or is that not needed?